dependency_injection/services.php

```diff
@@ -40,6 +40,12 @@
 return static function (ContainerConfigurator $container): void {
     $projectDir = dirname(__DIR__);
 
+    $parameters = $container->parameters();
+
+    // Default secret, just in case
+    $parameters->set('env(APP_SECRET_FILE)', $projectDir . '/config/glpicrypt.key');
+    $parameters->set('kernel.secret', env('file:APP_SECRET_FILE'));
```

Comment:
This file will not exist on a fresh installation, until its creation at the end of the installation process. What will be put in `kernel.secret` when the file does not exist ? Will it produce an error ?

Reply:
The parameter is used only in the Web Profiler (not even the debug toolbar), and Symfony will only fetch the variable at runtime (so if you open the Profiler). And you can't open the Profiler if the project isn't installed anyway...

Also, the installation process won't use the profiler, so it's safe for now.

Reply:
Then maybe defaulting to null is a better solution, if we want to be able, at some point, to use the profiler to debug the installation process.
```suggestion
    $parameters->set('kernel.secret', env('default::file:APP_SECRET_FILE'));
```

Reply:
Can't use `null`, it has to be a string. I created a default value that's stored during the container compilation process (then it happens only once)